control-plane/grpc/src/operations/volume/traits.rs

```diff
@@ -1656,6 +1675,27 @@ impl SetVolumeReplicaInfo for SetVolumeReplica {
     }
 }
 
+/// Trait to be implemented for SetVolumeProp operation.
+pub trait SetVolumePropInfo: Send + Sync + std::fmt::Debug {
+    /// Uuid of the concerned volume
+    fn uuid(&self) -> VolumeId;
+    /// Property name
+    fn prop_name(&self) -> String;
+    /// Property value
+    fn prop_value(&self) -> String;
+}
+
+impl SetVolumePropInfo for SetVolumeProp {
+    fn uuid(&self) -> VolumeId {
+        self.uuid.clone()
+    }
+    fn prop_name(&self) -> String {
+        self.prop_name.clone()
+    }
+    fn prop_value(&self) -> String {
+        self.prop_value.clone()
+    }
```

Comment:
As discussed probably should it be a map of params? Maybe for later

Reply:
hmm, you mean like set properties rather than set property?

Reply:
I mean yeah, accepting a map can set multiple properties at once. Like for example, max_snapshots and topology. Otherwise it would be multiple calls, which isn't bad either. Whatever is better ux.

Reply:
Currently created a new task for this. In the current PR, it is handled as property enum.
https://dcsw-bolt.atlassian.net/browse/GTM-1934